### ISSUE TITLE
Wrong format for bounding boxes

### ISSUE THREAD

**Issue by Rizhiy** [longcw/faster_rcnn_pytorch#36]
It seems that the network uses `x1,y1,x2,y2` format for bounding boxes instead of `x,y,w,h` used in the paper.
I think this is a pretty major difference that can affect training accuracy.

**Comment by Cadene** [comment]
Did you try it?

**Comment by Rizhiy** [comment]
I haven't since I don't quite understand the whole codebase and it appears that quite a bit would have to be changed. In particular, it appears that [cython code](https://github.com/longcw/faster_rcnn_pytorch/blob/master/faster_rcnn/rpn_msr/anchor_target_layer.py#L141) expects it in the current format and I don't have to access to cython source to change it.

It appears that this format was chosen in the fast-rcnn pytorch implementation or maybe even before, so probably would be difficult to change now.

**Comment by Cadene** [comment]
Yeap, unfortunately this code is difficult to understand and modify. As I was looking for some localization models in pytorch, I found this repo https://github.com/amdegroot/ssd.pytorch. The model works better than faster r-cnn and the codebase is way easier to understand.

It seems that the ssd.pytorch models use `x1,y1,x2,y2` format as weel https://github.com/amdegroot/ssd.pytorch/blob/master/data/voc0712.py#L81

**Comment by Rizhiy** [comment]
I found cython source, so might try to change it later.
